### ISSUE TITLE
Multiple slices with activation

### ISSUE THREAD

**Issue by KirstieJane** [KirstieJane/BrainsForPublication#10]
@roscha contributed code to create the following combined images of brain activation overlayed on multiple slices.

**Comment by KirstieJane** [comment]
Following on from discussions in the [#brains4publication](https://brainhack.slack.com/messages/brains4publication/) slack channel it might be good to minimise white space betweeen the brain images.

I'd also be interested to know if it's possible to explicitly set the number of brains you want on each row?

**Comment by roscha** [comment]
Yes you can set the number of brain and the number of row that you want ! 
I think the white space between brains is just dependent on the size of the figure, but I have to verify that and make it may be automatically well adjust.
